modules/client-configuration/pages/bootstrap-repository.adoc

```diff
@@ -91,22 +102,35 @@ mgr-create-bootstrap-repo -c SLE-version-x86_64
 The client repository is located in [path]``/srv/www/htdocs/pub/repositories/``.
 
 
-.Procedure: Specify a Bootstrap Repository
 
-If you have mirrored more than one {sle}{nbsp}15 Product (for example, SLES and SLES for SAP), you can specify the one you are actually interested in.
+.Procedure: Specifying a Parent Channel for a Bootstrap Repository
+
+If you have mirrored more than one product (for example, SLES and SLES for SAP), or if you use custom channels, you need to specify the parent channel to use when creating the bootstrap repository.
 
-. Check what bootstrap repositories you have available:
+. Check which parent channels you have available:
 +
 ----
-mgr-create-bootstrap-repo -c SLE-15-x86_64 --with-custom-channel
+mgr-create-bootstrap-repo -c SLE-15-x86_64
 Multiple options for parent channel found. Please use option
 --with-parent-channel <label> and choose one of:
 - sle-product-sles15-pool-x86_64
 - sle-product-sles_sap15-pool-x86_64
 - sle-product-sled15-pool-x86_64
 ----
-. Specify the appropriate repository:
+. Specify the appropriate parent channel:
 +
 ----
 mgr-create-bootstrap-repo -c SLE-15-x86_64 --with-parent-channel sle-product-sled15-pool-x86_64
 ----
+
+
+
+== Bootstrap and Custom Channels
+
+If you are using custom channels, you can use the [command]``--with-custom-channels`` option with the bootstrap command.
```

Comment:
```suggestion
If you are using custom channels, you can use the [option]``--with-custom-channels`` option with the [command]``mgr-create-bootstrap-repo`` command.
```
Assuming that we have the option tag and this bootstrap command is the right one.